.github/workflows/tr.unit.tests.yaml

```diff
@@ -56,8 +56,8 @@ jobs:
           key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
           restore-keys: |
             ${{ runner.os }}-maven-
-      - name: Run unit tests
-        uses: ./.github/actions/tr-tests
+      - name: Run Traffic Router unit tests
+        uses: ./.github/actions/tr-unit-tests
```

Comment:
I'm hoping to get the *TR Tests* GitHub Action to run the TR integration tests soon, so IMO we should leave it at *tr-tests* for now.